eslint.config.js

```diff
@@ -14,6 +14,5 @@ export default [
       },
     },
   },
-  plugins.n.configs["flat/recommended"],
```

Comment:
Need to confirm this change - lint doesn't pass as it complains localStorage isn't supported

Reply:
Try setting `env: ["browser"]` in neostandard config.

Reply:
@janfaracik please also remove `engines` from package.json. It was only required for this plugin.